### PR TITLE
Bug 1825064 - Change What's New URL to Fenix release notes

### PR DESCRIPTION
**Pull request by HarrisonOg**
### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/shared/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Breaking Changes**: If this is a breaking Android Components change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-apk-{fenix,focus,klar}-debug` task you're interested in.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1825064

### CODE REVIEW

**Comment by HarrisonOg** [comment]
> 

oh interesting, I didn't know that. I'll make the change.

File: fenix/app/src/main/java/org/mozilla/fenix/settings/SupportUtils.kt [hunk 1]
```diff
@@ -98,7 +98,13 @@ object SupportUtils {
         return "https://www.mozilla.org/$langTag/$path"
     }
 
-    fun getWhatsNewUrl(context: Context) = getSumoURLForTopic(context, SumoTopic.WHATS_NEW)
```
Comment (gabrielluong) [inline fenix/app/src/main/java/org/mozilla/fenix/settings/SupportUtils.kt:101]: Is `SumoTopic.WHATS_NEW` used anywhere or is the URL pointed by it relevant anymore after this change? Could we remove it if it isn't relevant anymore?

Comment (HarrisonOg) [reply fenix/app/src/main/java/org/mozilla/fenix/settings/SupportUtils.kt:101]: good catch, I've removed the unused enum and cleaned up the code a bit.

**Review by gabrielluong — APPROVED** [review]

**Comment by rvandermeulen** [comment]
The test fixup makes sense to me, but gabriel should probably be the one to re-review this if anyone.

**Review by gabrielluong — APPROVED** [review]

**Pull request merged**